float64 local_x
float64 local_y
uint32 height
uint32 az_angle
